src/cascade/core/context.py

```diff
@@ -17,6 +18,63 @@ class _ModelParameters:
     node = ParameterProperty()
 
 
+class _Rates:
+    __slots__ = ["pini", "iota", "rho", "chi", "omega"]
+
+    def __init__(self):
+        self.pini = Rate("pini")
+        self.iota = Rate("iota")
+        self.rho = Rate("rho")
+        self.chi = Rate("chi")
+        self.omega = Rate("omega")
+
+    def __iter__(self):
+        return iter([self.pini, self.iota, self.rho, self.chi, self.omega])
+
+
+class _Outputs:
+    __slots__ = ["integrands"]
+
+    def __init__(self):
+        self.integrands = _Integrands()
+
+
+class _Integrand:
+    def __init__(self, name):
+        self.name = name
+        self.active = False
```

Comment:
I don't know what makes it active so I am requiring that you explicitly set the flag. If you have a better way, I'm open.

Reply:
Where does the active flag get used? It's not for the integrands table. That's just a list of integrands. Maybe the active flag is never used?

Reply:
src/cascade/executor/no_covariate_main.py:211

It's used when the setup code interprets the user's non_zero_rates list. It's more than likely that I'm approaching this wrong, especially since you seem to find it unclear.